From 9ad8d634e50a39f132797984a7a726582d7b937d Mon Sep 17 00:00:00 2001
From: Evan Quan <evan.quan@amd.com>
Date: Tue, 6 Aug 2019 10:14:17 +0800
Subject: [PATCH 3307/4256] drm/amdkfd: fix missing break in kfd2kgd pointer
 init

This seems a cherry-pick error and will break Arcturus KFD.

Needed for hybrid branch only.

Affected ASIC: Arcturus

Change-Id: I4b4d845172144296624cb4c3255d440bcaef4705
Signed-off-by: Evan Quan <evan.quan@amd.com>
Reviewed-by: Le Ma <Le.Ma@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd.c
index 074206458323..8a1ba9c93e12 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd.c
@@ -96,6 +96,7 @@ void amdgpu_amdkfd_device_probe(struct amdgpu_device *adev)
 		break;
 	case CHIP_ARCTURUS:
 		kfd2kgd = amdgpu_amdkfd_arcturus_get_functions();
+		break;
 	case CHIP_NAVI10:
 	case CHIP_NAVI14:
 	case CHIP_NAVI12:
-- 
2.17.1

